idata_bytes() and resv_bytes() don't match their prototypes in common.c
which leads to a build error with watcom.  The following trivial patch
for 2.13.xx fixes it.

Reference: https://bugzilla.nasm.us/show_bug.cgi?id=3392409

diff --git a/common/common.c b/common/common.c
index 5a54620..fb3e890 100644
--- a/common/common.c
+++ b/common/common.c
@@ -66,7 +66,7 @@ const char *prefix_name(int token)
 /*
  * initialized data bytes length from opcode
  */
-int idata_bytes(int opcode)
+int idata_bytes(enum opcode opcode)
 {
     switch (opcode) {
     case I_DB:
@@ -95,7 +95,7 @@ int idata_bytes(int opcode)
 /*
  * Uninitialized data bytes length from opcode
  */
-int resv_bytes(int opcode)
+int resv_bytes(enum opcode opcode)
 {
     switch (opcode) {
     case I_RESB:
